src/core/toga/widgets/table.py

```diff
@@ -159,3 +195,6 @@ def on_select(self, handler):
         """
         self._on_select = wrapped_handler(self, handler)
         self._impl.set_on_select(self._on_select)
+
+    def refresh(self):
+        self._impl.refresh()
```

Comment:
Does the Interface level table need a refresh? Is there a reason the *user* would need to invoke `refresh()`? If not, should we just keep this on the implementation level?